website/content/docs/pipelines/config.mdx

```diff
@@ -0,0 +1,177 @@
+---
+layout: docs
+page_title: Pipelines Configuration
+description: |-
+  The page for detailing Waypoint pipeline configurations
+---
+
+# Pipelines Configuration
```

Comment:
Should we pluralize `pipeline(s)` or `configuration(s)`?

Reply:
Hm, how about `Pipeline Configurations`?

Reply:
Sounds good!